conversions/Forwards/src/main/java/com/agentdid127/resourcepack/forwards/impl/ImageFormatConverter.java

```diff
@@ -50,8 +51,18 @@ public void convert(Pack pack) throws IOException {
      * @throws IOException If the images are not found, or the remapping fails.
      */
     protected void findImage(Path path, String type) throws IOException {
+        // Find all files in the directory
         File directory = path.toFile();
-        for (File file : directory.listFiles()) {
+        if (!directory.exists() || !directory.isDirectory()) {
+            return;
+        }
+        File[] files = directory.listFiles();
+        if (files == null) {
+            return;
+        }
+
+        // Check each file
+        for (File file : files) {

```

Comment:
```suggestion
        if (!directory.exists() || !directory.isDirectory()) {
            return;
        }

        // Check each file
        for (File file : directory.listFiles()) {
```

Looking at the code, we know that the file is a directory, and it exists, so by listing files, it will always be a valid array. This case should be fine. If the directory is empty, it will just not run the for loop, which is the expected output.